README.md

```diff
@@ -122,7 +122,13 @@ Also, we also provide Optuna docker images on [DockerHub](https://hub.docker.com
 
 ## Contribution
 
-Any contributions to Optuna are welcome! When you send a pull request, please follow the
+Any contributions to Optuna are welcome!
+
+If you are a new contributor, please check [good first issues](https://github.com/optuna/optuna/labels/good%20first%20issue). These are simple and well-defined so that they can be good starting points for you to get familiar with the contribution workflow and other developers.
```

Comment:
Some suggestions on the phrasing.
```suggestion
If you are a new contributor, please check [good first issues](https://github.com/optuna/optuna/labels/good%20first%20issue). These are relatively simple, well-defined and are often good starting points for you to get familiar with the contribution workflow and other developers.
```

Reply:
This is more natural than the previous one. Thanks!